dev/preview/workflow/preview/preview.sh

```diff
@@ -30,5 +30,5 @@ ensure_gcloud_auth
 
 leeway run dev/preview:create-preview
 leeway run dev/preview:build
-previewctl install-context --gcp-service-account "${PREVIEW_ENV_DEV_SA_KEY_PATH}" --retry 30
+previewctl install-context --gcp-service-account "${PREVIEW_ENV_DEV_SA_KEY_PATH}" --timeout 10m
```

Comment:
You need to make a similar change [here](https://github.com/gitpod-io/gitpod/blob/7797eb50c151ff1b2cea25a681baf502d308daae/.werft/ide-integration-tests-startup.sh#L171) and [here](https://github.com/gitpod-io/gitpod/blob/7797eb50c151ff1b2cea25a681baf502d308daae/.werft/workspace-run-integration-tests.sh#L190)